README.md

```diff
@@ -63,6 +63,31 @@ If you are not using JupyterLab and you only want to install the Jupyter AI `%%a
 
     $ pip install jupyter-ai-magics
 
+You may need to install third-party packages, for example, to use some model providers and some file formats with Jupyter AI. For example, to use OpenAI models, in addition to your current install also run
+
+```
+pip install langchain-openai
+```
+
+To use models from Anthropic, install:
+
+```
+pip install langchain-anthropic
+```
+
+After these installs you should see the OpenAI and Anthropic models in the drop down list. If you are using Claude models on AWS Bedrock, you must also install
+
+```
+pip install langchain-aws
+```
+
+Similarly, install `langchain-<provider>` packages for other providers as well, listed [here](#model-providers).
+
+To handle all supported use cases, you can install every dependency, which will give you access to all models currently supported by `jupyter-ai`. To install every dependency, run the following command, and then restart JupyterLab:
+
+```
+pip install jupyter-ai[all]
+```
```

Comment:
I would add some more context here, reduce repetition in examples, and isolate this in its own section.

````suggestion
### Minimal installation via `pip` (optional)

Most model providers in Jupyter AI require a specific dependency to be installed before they are available for use. These are called _provider dependencies_. Provider dependencies are optional to Jupyter AI, meaning that Jupyter AI can be installed with or without any provider dependencies installed.

If a provider requires a dependency that is not installed, its models are not listed in the user interface which allows you to select a language model. This offers a way for users to control which models are available in your Jupyter AI environment.

For example, to install Jupyter AI with only added support for Anthropic models, run:

```
pip install jupyter-ai langchain-anthropic
```

For more information on model providers and which dependencies they require, see [the model provider table](https://jupyter-ai.readthedocs.io/en/latest/users/index.html#model-providers).
````

Reply:
Please also make this change in the user docs.

Reply:
@dlqqq I made changes to the docs as per your notes above. Adapted it a little bit so it flows better.